.gitignore

```diff
@@ -20,6 +20,7 @@ venv/
 *.exe
 *.syso
 *.log
+*.test
```

Comment:
This is a potential footgun: I wouldn't expect a .test file to be ignored by default.

Maybe add this to your global gitignore instead?

Reply:
sure